lua/nordic/groups/integrations.lua

```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.fg, bold = true }
+    G.TreeFileIcon = { fg = C.blue2 }
+    G.TreeFileNameOpened = { fg = C.fg }
+    G.TreeSpecialFile = { fg = C.magenta.bright }
+    G.TreeGitConflict = { fg = C.magenta.bright }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { fg = C.black0, bg = C.blue2, bold = true }
```

Comment:
I think this should maybe link to Winbar?

Reply:
the whole 4 lines link to Winbar right?

Reply:
or just the titlebar

Reply:
> is it just link = "Winbar", because I can't find anything with the name "Winbar" in this file

Yup, that should work:
https://github.com/AlexvZyl/nordic.nvim/blob/1ee4044077059b34eacd18d93613495a602c131f/lua/nordic/groups/native.lua#L179

For future reference if you click the search bar on GitHub via this pr it will search the current repos contents. You can do the same locally with something like ripgrep or telescopes built in ripgrep picker.